src/backend/parser/parse_clause.c

```diff
@@ -3383,6 +3385,9 @@ addTargetToSortList(ParseState *pstate, TargetEntry *tle,
 			case SORTBY_NULLS_DEFAULT:
 				/* NULLS FIRST is default for DESC; other way for ASC */
 				sortcl->nulls_first = reverse;
+				if (sort_nulls_first_hook){
+					sort_nulls_first_hook(sortcl, reverse);
```

Comment:
1. Please follow the coding convention about brackets.
2. nulls_first is a BBF-specific behavior so a better hook name could be `sort_nulls_default_hook`?

Reply:
How about sort_nullsfirst_default_hook ?

since it's a behavior controls whether nullsfirst is default or not ?